x-pack/plugins/enterprise_search/public/applications/shared/layout/nav.test.tsx

```diff
@@ -73,7 +73,7 @@ const baseNavItems = [
     items: [
       {
         'data-test-subj': 'searchSideNav-Playground',
-        href: '/app/enterprise_search/applications/playground',
+        href: '/app/search_playground',
```

Comment:
we should confirm the url path for this. Likely should be `/app/build/playground`